pkg/backend/redis/redis_test.go

```diff
@@ -107,10 +107,11 @@ var _ = Describe("Backend", func() {
 		ibackend, err := NewBackend(logger, backendconfig.Config{
 			BackendType: "redis",
 			Redis: &backendconfig.RedisConfig{
-				KeyPrefix:      "test",
-				Client:         client,
-				Backoff:        backoffConfig,
-				ChunkSizeInGet: 1000,
+				KeyPrefix:         "test",
+				Client:            client,
+				Backoff:           backoffConfig,
+				ChunkSizeInGet:    1000,
+				ChunkSizeInDelete: 1000,
```

Comment:
Would you mind adding tests for chunked deletion (large queue)??

Reply:
I added tests for chunked deletion.
https://github.com/pfnet-research/pftaskqueue/pull/44/commits/f2b8503401d9719b8cf674d5b4c53c79def46ce6